homeassistant/components/brother/sensor.py

```diff
@@ -83,26 +95,26 @@ def extra_state_attributes(self):
         return self._attrs
 
     @property
-    def icon(self):
+    def icon(self) -> str | None:
         """Return the icon."""
-        return SENSOR_TYPES[self.kind][ATTR_ICON]
+        return SENSOR_TYPES[self.kind]["icon"]
```

Comment:
Use `self._description` is these methods. That was the point.

Reply:
Yes, I realized it after i turning off the computer, I guess I was tired 😉 I'll change this tonight.